src/Components/Navigation/Search.js

```diff
@@ -4,9 +4,12 @@ import * as Scrivito from 'scrivito';
 const SearchBox = Scrivito.connect(class extends React.Component {
   constructor(props) {
     super(props);
+
     this.state = {
       q: '',
     };
+
+    this.inputRef = React.createRef();
```

Comment:
*  [ ] The React example is `this.textInput = React.createRef();`. I don't think suffixing the names with `Ref` is helpful. We should keep the names as they are.

Reply:
Quote from https://reactjs.org/blog/2018/03/29/react-v-16-3.html#createref-api :

```jsx
class MyComponent extends React.Component {
  constructor(props) {
    super(props);

    this.inputRef = React.createRef();
  }

  render() {
    return <input type="text" ref={this.inputRef} />;
  }

  componentDidMount() {
    this.inputRef.current.focus();
  }
}
```

I actually think that the `Ref` suffix makes it clearer, especially for users of that variable.

Reply:
I see, the docs aren't consistent. Fine with me. 👍